NOTICE.md

```diff
@@ -0,0 +1,4 @@
+## License notes
```

Comment:
nope, LICENSE only for project license, all other in Notice file (take a look to apache projects)